db/migrate/20221015094918_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,17 @@
+class AddConfirmableToDevise < ActiveRecord::Migration[7.0]
+  def up
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    # add_column :users, :unconfirmed_email, :string # Only if using reconfirmable
+    add_index :users, :confirmation_token, unique: true
+    User.update_all confirmed_at: DateTime.now
+  end
+  def down
+    remove_index :users, :confirmation_token
+    remove_columns :users, :confirmation_token, :confirmed_at, :confirmation_sent_at
+    # remove_columns :users, :unconfirmed_email # Only if using reconfirmable
+  end
+
+
+end
```

Comment:
- [ ] An error happens while running `rails db:migrate` because of the last migration "AddConfirmableToDevise" you added to your records. Kindly ensure that your migrations run successfully without errors.
I suggest removing this migration since the Confirmable columns already exist in the `AddDeviseToUsers` migration.

<img width="701" alt="image" src="https://user-images.githubusercontent.com/77190203/195988314-16f2370f-78fa-4457-a63e-1c6cae875737.png">